### ISSUE TITLE
FE app can't reach the BE

### ISSUE THREAD

**Issue by anton-kozlovsky** [soulless-viewer/shipments#1]
Seems like .env file should be added.
Also you might notice that date picker doesn't really work.

https://user-images.githubusercontent.com/55976782/169121112-4feba896-ab40-4141-8417-c1e990a0d254.mp4

**Comment by soulless-viewer** [comment]
@anton-kozlovsky 

It definitely worked in Firefox and Chrome on Linux as well as in Safari on iOS (as you can see in README).
![image](https://user-images.githubusercontent.com/29832584/169126691-06eec9ef-6dea-4221-96fd-46ae99948f4f.jpeg)

I'll check what's going on when macOS is used...

**Comment by soulless-viewer** [comment]
@anton-kozlovsky 
You're absolutely right, the issue is related to the environment variable. I forgot to mention this in the docs.

[Usage section](https://github.com/soulless-viewer/shipments/blob/master/README.md#usage) has been updated.

**Comment by soulless-viewer** [comment]
@anton-kozlovsky 

I found what's wrong with datetime picker:
The problem is that I'm trying to work with the native browser datetime picker programmatically _(using `.showPicker()`)_, but this ~obvious~ functionality hasn't yet been implemented in most browsers.

![pr15591 content dev mdn mozit cloud_en-US_docs_Web_API_HTMLInputElement_showPicker](https://user-images.githubusercontent.com/29832584/169247445-871b630c-e422-469d-aeee-adc8f83cfec9.png)


https://pr15591.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/HTMLInputElement/showPicker#browser_compatibility

[Known issues section](https://github.com/soulless-viewer/shipments/blob/master/README.md#known-issues) has been added

**Comment by anton-kozlovsky** [comment]
Yup, I'm aware if that. Thus, it is better not use such functionality but use a third-party package which already took care of it, implement your own component.

**Comment by anton-kozlovsky** [comment]
BTW seems like you have a typo in readme: datapicker --> datepicker.
